storage/cleanup.go

```diff
@@ -89,6 +89,11 @@ func (m *cleanupManager) Cleanup(t time.Time) error {
 			"encountered errors when cleaning up fileset files for %v: %v", t, err))
 	}
 
+	if err := m.deleteInactiveFilesetFiles(); err != nil {
```

Comment:
Maybe also address the case of deleting files for namespaces that are no longer active.

Reply:
Good catch, yeah we should do this.

Reply:
should I include this as part of this PR or separately?

Reply:
You could do this as a separate PR if you feel like it won't fundamentally change this code much?

Reply:
I think it would be a separate function to attach as a similar flow and the findDirectories function can be used in either case, just `DeleteInactiveNamespaces` which would probably then just become `DeleteInactiveFiles` instead of Filesets or Namespaces.

Reply:
Hey so I did some poking around, and it looks like when a namespace no longer has shards its deleted immediately after. I'm not sure if its necessary to do the deletion here, and if it is,

then I think 2 things need to happen

1) the deletion function is localized to the cleanup manager
2) namespaces don't try to delete associated files when they lose all their shards

would there be any case where a namespace has files that don't get deleted after (2) happens?

Reply:
I was more worried about the case where a namespace is removed from a database. It won't loose any of its shards when this happens, so there'd be zombie dirs/files we'd need to cleanup in the namespace's base directory.